markdown/ch2.markdown

```diff
@@ -105,6 +97,14 @@ We will use a Gaussian kernel to smooth out bumps in our data so we can get a
 better idea of the overall shape.
 
 ```python
+# Make all plots appear inline in the Jupyter notebook
+%matplotlib inline
+# Set custom plotting style
+import matplotlib.pyplot as plt
+plt.style.use('style/elegant.mplstyle')
```

Comment:
So we've decided to put this before the first plot, rather than before all Python code?